quarkus/admin/build.gradle.kts

```diff
@@ -37,6 +37,10 @@ dependencies {
   implementation(project(":polaris-quarkus-service"))
   implementation(project(":polaris-quarkus-defaults"))
 
+  if (project.hasProperty("eclipseLinkDeps")) {
```

Comment:
Since we are not including any JDBC drivers anymore; we now need this hook.

Reply:
Looks like a fair trade-off to me.